coveralls/api.py

```diff
@@ -47,18 +47,25 @@ def __init__(self, token_required=True, **kwargs):
             self.config['repo_token'] = repo_token
 
         if os.environ.get('TRAVIS'):
-            is_travis = True
+            is_travis_or_circle = True
             self.config['service_name'] = file_config.get('service_name', None) or 'travis-ci'
             self.config['service_job_id'] = os.environ.get('TRAVIS_JOB_ID')
+        elif os.environ.get('CI'):
```

Comment:
Should this be `CIRCLE` instead of `CI`? Might be a bit confusing otherwise, as Jenkins is a perfectly valid *CI* to be using.